addons/website_event/controllers/main.py

```diff
@@ -269,7 +269,10 @@ def _process_attendees_form(self, event, form_details):
         registrations = {}
         global_values = {}
         for key, value in form_details.items():
-            counter, attr_name = key.split('-', 1)
+            try:
+                counter, attr_name = key.split('-', 1)
```

Comment:
Hm.. can we pop the values we don't want to process instead? `form_details` should just be valid from the start

```py
if not recaptacha_ok:
     raise
post.pop('recaptcha_token_response', None)
```

Reply:
Originally that's what I did, but I encountered this override that called the method before again and failed:
https://github.com/odoo/odoo/blob/22f652ddb85de065208371aa21fed8c674687063/addons/website_event_sale/controllers/main.py#L69

I could've pop'ed there, too.. but I figured this way is safer, in case there are custom modules out there doing something similar.

If you still prefer pop, even if has to be done twice, I'll adjust

Reply:
Ah indeed we can't propagate from the super..
Would say we explicitly pop out that key inside `_process_attendees_form` then.

Otherwise at some point we'll add non-comma separated values and have some other issue.
(though hopefully we refactor this thing before then..)

Reply:
Gotcha, changes done